pw_mgr/password_manager.py

```diff
@@ -26,11 +21,11 @@ def main():
             break
         if mode == "view":
             print("\n")
-            view()
+            view(fer)
```

Comment:
This has me a little confused, what exactly does it do when you pass a variable to a function like this? Like, I see why this would work in this context, you're using the 'fer' variable to call the init_key function to load key.key etc etc, but I don't fully understand where or how the variable is applied to the code of the function it's getting passed to.